processing/src/test/java/org/apache/druid/segment/join/MapJoinableFactoryTest.java

```diff
@@ -83,6 +84,36 @@ public void testBuildDataSourceIsRegisteredShouldReturnJoinableFromFactory()
     EasyMock.replay(noopJoinableFactory);
     Optional<Joinable> joinable = target.build(noopDataSource, condition);
     Assert.assertEquals(mockJoinable, joinable.get());
+
+  }
+
+  @Test
+  public void testComputeJoinCacheKey()
+  {
+    Optional<byte[]> expected = Optional.of(new byte[]{1, 2, 3});
+    EasyMock.expect(noopJoinableFactory.computeJoinCacheKey(noopDataSource)).andReturn(expected);
+    EasyMock.replay(noopJoinableFactory);
+    Optional<byte[]> actual = target.computeJoinCacheKey(noopDataSource);
+    Assert.assertEquals(expected, actual);
```

Comment:
Should use `Assert.assertArrayEquals()`.

Reply:
Replaced with `Assert.assertSame`